tests/integration/targets/filter_list/runme.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
I think this file and `runme.yml` can be deleted. They don't seem to achieve anything.

Reply:
Don't delete it. It's a playbook used in *runme.sh*

Reply:
Yeah, but why? Integration tests don't need a runme.sh and a playbook. That's only useful if you need to install something before the tests run, set environment variables for `ansible-playbook`, etc. For tests that do nothing special, using ansible-test's native way is a lot cleaner.

Reply:
OK. I can live without them.

Reply:
The scripts in the other roles are there because they cannot simply be used as roles, but some setup has to be done (like installing a Python dependency) that cannot be done in a role where the plugin is used (since plugins are loaded on startup, and most of them check their dependencies on load time). All other tests without a runme.sh are already simple roles.